packages/utils/type-test.ts

```diff
@@ -1,26 +0,0 @@
-import firebase from '@react-native-firebase/app';
```

Comment:
Could do with keeping this file but can move it into the root of `packages/app` without issues.

These `type-test` files are here just to validate the module usages work as expected when TSC runs over the project.

Could do with adding some lines on this as well that use some of the `FilePath` statics as well, just to make sure.